pkg/sql/plan/utils.go

```diff
@@ -655,3 +655,53 @@ func isConstant(e *plan.Expr) bool {
 		return false
 	}
 }
+
```

Comment:
These code, if you really need it, should go into container types.

Reply:
Ok, no problem